Dockerfile

```diff
@@ -1,7 +1,7 @@
 ARG GO_VERSION=1.17
 ARG BATS_VERSION=v1.3.0
 
-FROM golang:${GO_VERSION}-buster
+FROM golang:${GO_VERSION}-bullseye
 ARG DEBIAN_FRONTEND=noninteractive
 
 RUN echo 'deb https://download.opensuse.org/repositories/devel:/tools:/criu/Debian_10/ /' > /etc/apt/sources.list.d/criu.list \
```

Comment:
I guess this line needs to be updated too, but I don't see `Debian_11` repo there, so we can update later.

Reply:
Yes, I took a look, there's no Debian_11 repo and the existing one works fine.

@adrianreber hope you can add it later (either at the time of 3.16 release, or earlier).